packages/xy-chart/src/index.js

```diff
@@ -25,3 +25,4 @@ export { PatternLines, PatternCircles, PatternWaves, PatternHexagons } from '@vx
 export { withScreenSize, withParentSize, ParentSize } from '@vx/responsive';
 export { default as withTheme } from './enhancer/withTheme';
 export { chartTheme as theme } from '@data-ui/theme';
+export { default as Brush } from './selection/Brush';
```

Comment:
do you think all of the brush components should go under `src/components/brush/...` and the utils could go under `src/util/brush`?

Reply:
So here,  I was thinking for a long term, will we keep the code here or we should move it to vx? The reason I put it in the utils is that I think now it is the temporary solution.